lib/migrate/Migrator.js

```diff
@@ -278,10 +278,18 @@ class Migrator {
   forceFreeMigrationsLock(config) {
     this.config = getMergedConfig(config, this.config);
 
-    const lockTable = getLockTableName(this.config.tableName);
+    const lockTableName = getLockTableName(this.config.tableName);
+
     return getSchemaBuilder(this.knex, this.config.schemaName)
       .hasTable(lockTable)
-      .then((exist) => exist && this._freeLock());
+      .then((exist) => {
+        if (exist) {
+          await getTable(this.knex, lockTableName, this.config.schemaName).del();
```

Comment:
`this.config.schemaName` and `lockTableName` usage is not consistent.

Reply:
@kibertoad Can you explain what you mean by `not consistent`? `lockTableName` is used in multiple locations in the outer function, which is why i assigned it to a `const`.

Reply:
yeah, this is fair. still for symmetry sake would be nice to do `const lockSchemaName = ` as well.